GithubBrowserSample/app/src/main/java/com/android/example/github/ui/user/UserFragment.kt

```diff
@@ -70,13 +80,32 @@ class UserFragment : Fragment(), Injectable {
             }
         }
         binding = dataBinding
+        sharedElementEnterTransition = TransitionInflater.from(context).inflateTransition(R.transition.move)
+        // When the image is loaded, set the image request listener to start the transaction
+        binding.imageRequestListener = object: RequestListener<Drawable> {
+            override fun onLoadFailed(e: GlideException?, model: Any?, target: Target<Drawable>?, isFirstResource: Boolean): Boolean {
+                startPostponedEnterTransition()
+                return false
+            }
+
+            override fun onResourceReady(resource: Drawable?, model: Any?, target: Target<Drawable>?, dataSource: DataSource?, isFirstResource: Boolean): Boolean {
+                startPostponedEnterTransition()
+                return false
+            }
+        }
+        // Animation Watchdog - Make sure we don't wait longer than a second for the Glide image
+        handler.postDelayed(1000) {
+            startPostponedEnterTransition()
```

Comment:
add a check for `lifecycle.isAtLeastStarted` to avoid any fragment transaction error (actually idk if this would cause a transaction error so feel free to ignore this comment)

Reply:
I did some spelunking through the fragment code, and I don't see any way this would generate a fragment transaction error.